src/inc/clrconfigvalues.h

```diff
@@ -450,8 +450,8 @@ RETAIL_CONFIG_STRING_INFO_EX(INTERNAL_AltJitNgen, W("AltJitNgen"), "Enables AltJ
 #endif // defined(ALLOW_SXS_JIT_NGEN)
 CONFIG_DWORD_INFO_EX(INTERNAL_JitNoCMOV, W("JitNoCMOV"), 0, "", CLRConfig::REGUTIL_default)
 
-CONFIG_STRING_INFO_EX(INTERNAL_JitValNumCSE,  W("JitValNumCSE"),  "Enables ValNum CSE for the specified methods", CLRConfig::REGUTIL_default) 
-CONFIG_STRING_INFO_EX(INTERNAL_JitLexicalCSE, W("JitLexicalCSE"), "Enables Lexical CSE for the specified methods", CLRConfig::REGUTIL_default) 
+CONFIG_STRING_INFO_EX(INTERNAL_JitValNumCSE,  W("JitValNumCSE"),  "Enables ValNum CSE for the specified methods", CLRConfig::REGUTIL_default)
```

Comment:
The switches that are covered by `src\jit\jitconfigvalues.h` can be deleted from this file.

Reply:
@jkotas There are 142 different switches which are defined in both `clrconfigvalues.h` and `jitconfigvalues.h`, deleting all from `clrconfigvalues.h`

```
Duplicate: clrconfigvalues.h:354   DumpJittedMethods -> jitconfigvalues.h:29   DumpJittedMethods
Duplicate: clrconfigvalues.h:357   JitCloneLoops -> jitconfigvalues.h:41 JitCloneLoops
Duplicate: clrconfigvalues.h:358   JitAssertOnMaxRAPasses -> jitconfigvalues.h:33 JitAssertOnMaxRAPasses
Duplicate: clrconfigvalues.h:359   JitBreak   -> jitconfigvalues.h:143 JitBreak
Duplicate: clrconfigvalues.h:361   JitBreakEmitOutputInstr -> jitconfigvalues.h:34 JitBreakEmitOutputInstr
Duplicate: clrconfigvalues.h:362   JitBreakMorphTree -> jitconfigvalues.h:35 JitBreakMorphTree
Duplicate: clrconfigvalues.h:363   JitBreakOnBadCode -> jitconfigvalues.h:36 JitBreakOnBadCode
Duplicate: clrconfigvalues.h:364   JitBreakOnUnsafeCode -> jitconfigvalues.h:38 JitBreakOnUnsafeCode
Duplicate: clrconfigvalues.h:365   JitCanUseSSE2 -> jitconfigvalues.h:39 JitCanUseSSE2
Duplicate: clrconfigvalues.h:366   JitDebugBreak -> jitconfigvalues.h:143 JitDebugBreak
Duplicate: clrconfigvalues.h:368   JitDefaultFill -> jitconfigvalues.h:45 JitDefaultFill
Duplicate: clrconfigvalues.h:369   JitDirectAlloc -> jitconfigvalues.h:45 JitDirectAlloc
Duplicate: clrconfigvalues.h:375   JitEnableNoWayAssert -> jitconfigvalues.h:260 JitEnableNoWayAssert
Duplicate: clrconfigvalues.h:375   JitEnableNoWayAssert -> jitconfigvalues.h:262 JitEnableNoWayAssert
Duplicate: clrconfigvalues.h:376   JitDisasm -> jitconfigvalues.h:145 JitDisasm
Duplicate: clrconfigvalues.h:377   JitDoubleAlign -> jitconfigvalues.h:46 JitDoubleAlign
Duplicate: clrconfigvalues.h:378   JitDump   -> jitconfigvalues.h:146 JitDump
Duplicate: clrconfigvalues.h:379   JitDumpIR -> jitconfigvalues.h:147 JitDumpIR
Duplicate: clrconfigvalues.h:380   JitDumpIRFormat -> jitconfigvalues.h:185 JitDumpIRFormat
Duplicate: clrconfigvalues.h:381   JitDumpIRPhase -> jitconfigvalues.h:186 JitDumpIRPhase
Duplicate: clrconfigvalues.h:382   JitDumpVerboseTrees -> jitconfigvalues.h:53 JitDumpVerboseTrees
Duplicate: clrconfigvalues.h:383   JitDumpVerboseSsa -> jitconfigvalues.h:52 JitDumpVerboseSsa
Duplicate: clrconfigvalues.h:384   JitDumpBeforeAfterMorph -> jitconfigvalues.h:142 JitDumpBeforeAfterMorph
Duplicate: clrconfigvalues.h:385   JitDumpFg -> jitconfigvalues.h:176 JitDumpFg
Duplicate: clrconfigvalues.h:386   JitDumpFgDir -> jitconfigvalues.h:177 JitDumpFgDir
Duplicate: clrconfigvalues.h:387   JitDumpFgFile -> jitconfigvalues.h:178 JitDumpFgFile
Duplicate: clrconfigvalues.h:388   JitDumpFgPhase -> jitconfigvalues.h:182 JitDumpFgPhase
Duplicate: clrconfigvalues.h:389   JitDumpFgDot -> jitconfigvalues.h:49 JitDumpFgDot
Duplicate: clrconfigvalues.h:391   JitDumpASCII -> jitconfigvalues.h:48 JitDumpASCII
Duplicate: clrconfigvalues.h:392   JitDumpTerseLsra -> jitconfigvalues.h:50 JitDumpTerseLsra
Duplicate: clrconfigvalues.h:393   JitDumpToDebugger -> jitconfigvalues.h:51 JitDumpToDebugger
Duplicate: clrconfigvalues.h:394   JitEmitPrintRefRegs -> jitconfigvalues.h:53 JitEmitPrintRefRegs
Duplicate: clrconfigvalues.h:395   JitExclude -> jitconfigvalues.h:148 JitExclude
Duplicate: clrconfigvalues.h:396   JitForceFallback -> jitconfigvalues.h:61 JitForceFallback
Duplicate: clrconfigvalues.h:397   JitNoForceFallback -> jitconfigvalues.h:93 JitNoForceFallback
Duplicate: clrconfigvalues.h:398   JitExpensiveDebugCheckLevel -> jitconfigvalues.h:60   JitExpensiveDebugCheckLevel
Duplicate: clrconfigvalues.h:399   JitForceProcedureSplitting -> jitconfigvalues.h:149   JitForceProcedureSplitting
Duplicate: clrconfigvalues.h:400   JitForceVer -> jitconfigvalues.h:61 JitForceVer
Duplicate: clrconfigvalues.h:402   JitFullyInt -> jitconfigvalues.h:63 JitFullyInt
Duplicate: clrconfigvalues.h:403   JitGCChecks -> jitconfigvalues.h:64 JitGCChecks
Duplicate: clrconfigvalues.h:404   JitGCDump -> jitconfigvalues.h:150 JitGCDump
Duplicate: clrconfigvalues.h:405   JitGCInfoLogging -> jitconfigvalues.h:66 JitGCInfoLogging
Duplicate: clrconfigvalues.h:407   JitHalt   -> jitconfigvalues.h:152 JitHalt
Duplicate: clrconfigvalues.h:408   JitHashHalt -> jitconfigvalues.h:70 JitHashHalt
Duplicate: clrconfigvalues.h:409   JitHashBreak -> jitconfigvalues.h:67 JitHashBreak
Duplicate: clrconfigvalues.h:410   JitHashDump -> jitconfigvalues.h:68 JitHashDump
Duplicate: clrconfigvalues.h:411   JitHashDumpIR -> jitconfigvalues.h:69 JitHashDumpIR
Duplicate: clrconfigvalues.h:414   JitImportBreak -> jitconfigvalues.h:152 JitImportBreak
Duplicate: clrconfigvalues.h:415   JitInclude -> jitconfigvalues.h:153 JitInclude
Duplicate: clrconfigvalues.h:416   JitInlineAdditionalMultiplier -> jitconfigvalues.h:70   JitInlineAdditionalMultiplier
Duplicate: clrconfigvalues.h:417   JitInlineSIMDMultiplier -> jitconfigvalues.h:287 JitInlineSIMDMultiplier
Duplicate: clrconfigvalues.h:418   JitInlinePrintStats -> jitconfigvalues.h:71 JitInlinePrintStats
Duplicate: clrconfigvalues.h:419   JITInlineSize -> jitconfigvalues.h:72 JITInlineSize
Duplicate: clrconfigvalues.h:420   JitLateDisasm -> jitconfigvalues.h:154 JitLateDisasm
Duplicate: clrconfigvalues.h:421   JITLateDisasmTo -> jitconfigvalues.h:186 JITLateDisasmTo
Duplicate: clrconfigvalues.h:422   JITMaxTempAssert -> jitconfigvalues.h:75 JITMaxTempAssert
Duplicate: clrconfigvalues.h:423   JitMaxUncheckedOffset -> jitconfigvalues.h:76 JitMaxUncheckedOffset
Duplicate: clrconfigvalues.h:424   JITMinOpts -> jitconfigvalues.h:78 JITMinOpts
Duplicate: clrconfigvalues.h:425   JITMinOptsBbCount -> jitconfigvalues.h:79 JITMinOptsBbCount
Duplicate: clrconfigvalues.h:426   JITMinOptsCodeSize -> jitconfigvalues.h:81 JITMinOptsCodeSize
Duplicate: clrconfigvalues.h:427   JITMinOptsInstrCount -> jitconfigvalues.h:83 JITMinOptsInstrCount
Duplicate: clrconfigvalues.h:428   JITMinOptsLvNumcount -> jitconfigvalues.h:85 JITMinOptsLvNumcount
Duplicate: clrconfigvalues.h:429   JITMinOptsLvRefcount -> jitconfigvalues.h:87 JITMinOptsLvRefcount
Duplicate: clrconfigvalues.h:430   JITBreakOnMinOpts -> jitconfigvalues.h:38 JITBreakOnMinOpts
Duplicate: clrconfigvalues.h:431   JITMinOptsName -> jitconfigvalues.h:156 JITMinOptsName
Duplicate: clrconfigvalues.h:435   AltJit   -> jitconfigvalues.h:314 AltJit
Duplicate: clrconfigvalues.h:437   AltJitLimit -> jitconfigvalues.h:18 AltJitLimit
Duplicate: clrconfigvalues.h:438   RunAltJitCode -> jitconfigvalues.h:136 RunAltJitCode
Duplicate: clrconfigvalues.h:451   JitNoCMOV -> jitconfigvalues.h:87 JitNoCMOV
Duplicate: clrconfigvalues.h:455   JitNoCSE   -> jitconfigvalues.h:88 JitNoCSE
Duplicate: clrconfigvalues.h:456   JitNoCSE2 -> jitconfigvalues.h:89 JitNoCSE2
Duplicate: clrconfigvalues.h:457   JitNoHoist -> jitconfigvalues.h:93 JitNoHoist
Duplicate: clrconfigvalues.h:459   JitNoInline -> jitconfigvalues.h:95 JitNoInline
Duplicate: clrconfigvalues.h:460   JitAggressiveInlining -> jitconfigvalues.h:286 JitAggressiveInlining
Duplicate: clrconfigvalues.h:462   JitNoProcedureSplitting -> jitconfigvalues.h:158 JitNoProcedureSplitting
Duplicate: clrconfigvalues.h:463   JitNoProcedureSplittingEH -> jitconfigvalues.h:161   JitNoProcedureSplittingEH
Duplicate: clrconfigvalues.h:464   JitNoRegLoc -> jitconfigvalues.h:96 JitNoRegLoc
Duplicate: clrconfigvalues.h:465   JitNoStructPromotion -> jitconfigvalues.h:98 JitNoStructPromotion
Duplicate: clrconfigvalues.h:466   JitNoUnroll -> jitconfigvalues.h:98 JitNoUnroll
Duplicate: clrconfigvalues.h:467   JitNoMemoryBarriers -> jitconfigvalues.h:96 JitNoMemoryBarriers
Duplicate: clrconfigvalues.h:469   JitNoRngChks -> jitconfigvalues.h:232 JitNoRngChks
Duplicate: clrconfigvalues.h:469   JitNoRngChks -> jitconfigvalues.h:291 JitNoRngChks
Duplicate: clrconfigvalues.h:472   JitOrder   -> jitconfigvalues.h:99 JitOrder
Duplicate: clrconfigvalues.h:473   JitDiffableDasm -> jitconfigvalues.h:26 JitDiffableDasm
Duplicate: clrconfigvalues.h:474   JitSlowDebugChecksEnabled -> jitconfigvalues.h:110   JitSlowDebugChecksEnabled
Duplicate: clrconfigvalues.h:475   JITPInvokeCheckEnabled -> jitconfigvalues.h:101 JITPInvokeCheckEnabled
Duplicate: clrconfigvalues.h:476   JITPInvokeEnabled -> jitconfigvalues.h:102 JITPInvokeEnabled
Duplicate: clrconfigvalues.h:477   JitPrintInlinedMethods -> jitconfigvalues.h:103 JitPrintInlinedMethods
Duplicate: clrconfigvalues.h:478   JitTelemetry -> jitconfigvalues.h:308 JitTelemetry
Duplicate: clrconfigvalues.h:479   JitRange   -> jitconfigvalues.h:187 JitRange
Duplicate: clrconfigvalues.h:480   JITRequired -> jitconfigvalues.h:105 JITRequired
Duplicate: clrconfigvalues.h:481   JITRoundFloat -> jitconfigvalues.h:106 JITRoundFloat
Duplicate: clrconfigvalues.h:482   JitSkipArrayBoundCheck -> jitconfigvalues.h:108 JitSkipArrayBoundCheck
Duplicate: clrconfigvalues.h:483   JitStackChecks -> jitconfigvalues.h:115 JitStackChecks
Duplicate: clrconfigvalues.h:484   JitStackProbes -> jitconfigvalues.h:138 JitStackProbes
Duplicate: clrconfigvalues.h:485   JitStress -> jitconfigvalues.h:118 JitStress
Duplicate: clrconfigvalues.h:486   JitStressBBProf -> jitconfigvalues.h:119 JitStressBBProf
Duplicate: clrconfigvalues.h:487   JitStressFP -> jitconfigvalues.h:124 JitStressFP
Duplicate: clrconfigvalues.h:488   JitStressModeNames -> jitconfigvalues.h:190 JitStressModeNames
Duplicate: clrconfigvalues.h:489   JitStressModeNamesNot -> jitconfigvalues.h:193 JitStressModeNamesNot
Duplicate: clrconfigvalues.h:490   JitStressOnly -> jitconfigvalues.h:162 JitStressOnly
Duplicate: clrconfigvalues.h:491   JitStressRange -> jitconfigvalues.h:194 JitStressRange
Duplicate: clrconfigvalues.h:492   JitStressRegs -> jitconfigvalues.h:126 JitStressRegs
Duplicate: clrconfigvalues.h:493   JitStressBiasedCSE -> jitconfigvalues.h:123 JitStressBiasedCSE
Duplicate: clrconfigvalues.h:494   JitStrictCheckForNonVirtualCallToVirtualMethod -> jitconfigvalues.h:127   JitStrictCheckForNonVirtualCallToVirtualMethod
Duplicate: clrconfigvalues.h:496   JitTimeLogCsv -> jitconfigvalues.h:328 JitTimeLogCsv
Duplicate: clrconfigvalues.h:497   JitFuncInfoLogFile -> jitconfigvalues.h:326 JitFuncInfoLogFile
Duplicate: clrconfigvalues.h:498   JitUnwindDump -> jitconfigvalues.h:163 JitUnwindDump
Duplicate: clrconfigvalues.h:499   JitEHDump -> jitconfigvalues.h:148 JitEHDump
Duplicate: clrconfigvalues.h:503   TailCallOpt -> jitconfigvalues.h:328 TailCallOpt
Duplicate: clrconfigvalues.h:504   TailcallStress -> jitconfigvalues.h:140 TailcallStress
Duplicate: clrconfigvalues.h:505   TailCallLoopOpt -> jitconfigvalues.h:313 TailCallLoopOpt
Duplicate: clrconfigvalues.h:507   JitDoSsa   -> jitconfigvalues.h:300 JitDoSsa
Duplicate: clrconfigvalues.h:508   JitDoEarlyProp -> jitconfigvalues.h:297 JitDoEarlyProp
Duplicate: clrconfigvalues.h:509   JitDoValueNumber -> jitconfigvalues.h:301 JitDoValueNumber
Duplicate: clrconfigvalues.h:510   JitDoLoopHoisting -> jitconfigvalues.h:298 JitDoLoopHoisting
Duplicate: clrconfigvalues.h:511   JitDoCopyProp -> jitconfigvalues.h:296 JitDoCopyProp
Duplicate: clrconfigvalues.h:512   JitDoAssertionProp -> jitconfigvalues.h:295 JitDoAssertionProp
Duplicate: clrconfigvalues.h:513   JitDoRangeAnalysis -> jitconfigvalues.h:299 JitDoRangeAnalysis
Duplicate: clrconfigvalues.h:514   JitSsaStress -> jitconfigvalues.h:115 JitSsaStress
Duplicate: clrconfigvalues.h:517   AltJitAssertOnNYI -> jitconfigvalues.h:238 AltJitAssertOnNYI
Duplicate: clrconfigvalues.h:517   AltJitAssertOnNYI -> jitconfigvalues.h:240 AltJitAssertOnNYI
Duplicate: clrconfigvalues.h:521   AltJitSkipOnAssert -> jitconfigvalues.h:21 AltJitSkipOnAssert
Duplicate: clrconfigvalues.h:523   JitSplitFunctionSize -> jitconfigvalues.h:113 JitSplitFunctionSize
Duplicate: clrconfigvalues.h:524   JitRegisterFP -> jitconfigvalues.h:307 JitRegisterFP
Duplicate: clrconfigvalues.h:525   JitELTHookEnabled -> jitconfigvalues.h:287 JitELTHookEnabled
Duplicate: clrconfigvalues.h:526   JitComponentUnitTests -> jitconfigvalues.h:137 JitComponentUnitTests
Duplicate: clrconfigvalues.h:527   JitMemStats -> jitconfigvalues.h:284 JitMemStats
Duplicate: clrconfigvalues.h:528   JitLoopHoistStats -> jitconfigvalues.h:27 JitLoopHoistStats
Duplicate: clrconfigvalues.h:529   JitDebugLogLoopCloning -> jitconfigvalues.h:43 JitDebugLogLoopCloning
Duplicate: clrconfigvalues.h:530   JitVNMapSelLimit -> jitconfigvalues.h:130 JitVNMapSelLimit
Duplicate: clrconfigvalues.h:531   JitVNMapSelBudget -> jitconfigvalues.h:310 JitVNMapSelBudget
Duplicate: clrconfigvalues.h:547   JitEnablePCRelAddr -> jitconfigvalues.h:31 JitEnablePCRelAddr
Duplicate: clrconfigvalues.h:557   JitFunctionTrace -> jitconfigvalues.h:64 JitFunctionTrace
Duplicate: clrconfigvalues.h:684   NgenDisasm -> jitconfigvalues.h:167 NgenDisasm
Duplicate: clrconfigvalues.h:685   NgenDump   -> jitconfigvalues.h:168 NgenDump
Duplicate: clrconfigvalues.h:686   NgenDumpIR -> jitconfigvalues.h:169 NgenDumpIR
Duplicate: clrconfigvalues.h:687   NgenDumpIRFormat -> jitconfigvalues.h:201 NgenDumpIRFormat
Duplicate: clrconfigvalues.h:688   NgenDumpIRPhase -> jitconfigvalues.h:202 NgenDumpIRPhase
Duplicate: clrconfigvalues.h:689   NgenDumpFg -> jitconfigvalues.h:198 NgenDumpFg
Duplicate: clrconfigvalues.h:690   NgenDumpFgDir -> jitconfigvalues.h:199 NgenDumpFgDir
Duplicate: clrconfigvalues.h:691   NgenDumpFgFile -> jitconfigvalues.h:200 NgenDumpFgFile
Duplicate: clrconfigvalues.h:693   NgenGCDump -> jitconfigvalues.h:170 NgenGCDump
Duplicate: clrconfigvalues.h:708   NgenUnwindDump -> jitconfigvalues.h:172 NgenUnwindDump
Duplicate: clrconfigvalues.h:709   NgenEHDump -> jitconfigvalues.h:170 NgenEHDump

```